new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -375,6 +380,8 @@ public void testDomainInPvUsingWlst() {
         .withFailMessage("NGINX can not access the test web app from one or more of the managed servers")
         .isTrue();
 
+    assertTrue(shutdownDomain(domainUid, wlstDomainNamespace), "Failed to shutdown domain");
```

Comment:
Initially I added the tests in ItDominInPV.java and these tests were running after the first 2 which created 4 server pods. In order not to strain the cluster I thought of shutting down the domain before the introspectVersion test runs.

Since the introspectVersion tests are moved to a different class I removed these shutdown calls.